static/_redirects

```diff
@@ -0,0 +1,2 @@
+/js/script.js https://plausible.io/js/plausible.js 200
```

Comment:
A renommer autre chose que "script.js"

Reply:
Vaut mieux que ça soit générique si on veut pas se faire bloquer

Reply:
https://plausible.io/docs/proxy/guides/netlify

> Choose a generic or irrelevant name for the subdirectory. If you choose something like analytics or plausible, it might get blocked in the future.